_pages/about-us.md

```diff
@@ -7,5 +7,15 @@ header:
  overlay_filter: "1"
 excerpt: "Learn more about OpenPrinting"
 ---
+## A Brief History
 
-Page under construction
\ No newline at end of file
+OpenPrinting Workgroup was initially established under Free Standards Group(FSG). In July 2006 Linuxprinting.org merged with FSG's OP WorkGroup. On January 22, 2007, the FSG and the Open Source Development Labs(OSDL) merged to form **The Linux Foundation**. Now, OpenPrinting is an opensource organization under The Linux Foundation.
```

Comment:
Use "... is an free software organization ..." instead of "... is an opensource organization ...". Also generally do not write "opensource" as one word, but write "open source".

Reply:
The issue has been corrected.